community/kernel/src/main/java/org/neo4j/graphdb/factory/GraphDatabaseSettings.java

```diff
@@ -838,6 +838,13 @@ public String providerName()
     public static final Setting<String> default_advertised_address =
             setting( "dbms.connectors.default_advertised_address", STRING, "localhost" );
 
+    @Description( "Default precision of temporal time based values. Corresponds to the number of value " +
+            "figures in the nanosecond part, i.e. 0 corresponds to second , 3 to millisecond, 6 to " +
```

Comment:
I think it would read better if `number of value figures` was replaced by `number of decimal points`.